src/backend/dx11/src/device.rs

```diff
@@ -2461,6 +2461,30 @@ impl device::Device<Backend> for Device {
         // TODO
     }
 
+    unsafe fn set_display_power_state(
```

Comment:
weird to see these changes here. Wrong rebase?

Reply:
There was some conflicts that makes the github merge check to fail, so i have merged it with the master branch and solved the conflicts. Shouldn't I have?

Reply:
Please rebase instead of merging

Reply:
I'm sorry for the inconvenience, i hope i made the rebase correctly. If not, please notify me and will fix it as soon as possible.